deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/wazuh_deployment/WazuhDeployment.py

```diff
@@ -64,7 +65,7 @@ def install(self, install_type):
 
             tasks_list.append(AnsibleTask({
                 'name': 'Executing "install.sh" script to build and install Wazuh',
-                'shell': './install.sh > /tmp/wazuh_install_log.txt',
+                'shell': f'./install.sh > {gettempdir()}/wazuh_install_log.txt',
```

Comment:
```suggestion
                'shell': f"./install.sh > {gettempdir()}/wazuh_install_log.txt",
```

Reply:
done in 
https://github.com/wazuh/wazuh-qa/pull/1715/commits/0e86bd224c659e238be3973a4bad443e4cb22c14